### PR TITLE
apply quarkus changes to openshift getting started guide

### PR DESCRIPTION
**Pull request by DGuhr**
closes keycloak/keycloak#10569

### CODE REVIEW

File: guides/getting-started/getting-started-openshift.adoc [hunk 1]
```diff
@@ -56,14 +56,13 @@ Once the command above completes you should see a message similar to this:
 
 [source,bash,subs="attributes+"]
 ----
-service/secure-keycloak created
+service/keycloak created
```
Comment (xingxingxia) [inline guides/getting-started/getting-started-openshift.adoc:59]: Same as https://github.com/keycloak/keycloak-quickstarts/pull/304#discussion_r822275265 , using insecure way is not recommended in modern time. Hope we can keep secure way while fixing other issue in question.

**Review by andreaTP — APPROVED** [review]

File: guides/getting-started/getting-started-openshift.adoc [hunk 2]
```diff
@@ -46,8 +46,8 @@ To spin up a Keycloak server in your project, execute the following command:
 [source,bash,subs="attributes+"]
 ----
 oc process -f https://raw.githubusercontent.com/keycloak/keycloak-quickstarts/latest/openshift-examples/keycloak.yaml \
-    -p KEYCLOAK_USER=admin \
-    -p KEYCLOAK_PASSWORD=admin \
+    -p KEYCLOAK_ADMIN=admin \
+    -p KEYCLOAK_ADMIN_PASSWORD=admin \
```
Comment (vmuzikar) [inline guides/getting-started/getting-started-openshift.adoc:50]: AFAIK, we're not using H2 by default, so this won't work. DB has to be explicitly set.

**Review by vmuzikar — CHANGES_REQUESTED** [review]

File: guides/getting-started/getting-started-openshift.adoc (hunk 2 above)
Comment (andreaTP) [reply guides/getting-started/getting-started-openshift.adoc:50]: sync with comments on:
https://github.com/keycloak/keycloak-quickstarts/pull/304

File: guides/getting-started/getting-started-openshift.adoc (hunk 1 above)
[the remark this answers is not in the document]
Comment (andreaTP) [reply guides/getting-started/getting-started-openshift.adoc:59]: no, probably a typo or an already outdated log copy paste

File: guides/getting-started/getting-started-openshift.adoc (hunk 2 above)
Comment (andreaTP) [reply guides/getting-started/getting-started-openshift.adoc:50]: don and it doesn't require changes in this PR

**Review by andreaTP — APPROVED** [review]
@vmuzikar this PR is ready to be merged after https://github.com/keycloak/keycloak-quickstarts/pull/306

**Review by vmuzikar — APPROVED** [review]

**Review by abstractj — APPROVED** [review]

**Pull request merged**